lib/std/os/linux.zig

```diff
@@ -219,6 +219,10 @@ pub fn mmap(address: ?[*]u8, length: usize, prot: usize, flags: u32, fd: i32, of
     }
 }
 
+pub fn fcntl(fd: fd_t, cmd: i32, flock_p: *const c_void) usize {
```

Comment:
`fcntl` is not used exclusively for file-locking purposes, let's follow the libc and use a neutral `arg`.
On top of that the `arg` parameter is optional (some ops don't require it) and can be written to (eg. if you use `F_GETLK`) so I'd say `?*c_void` is better here.